File: swift/nb.swift
import Foundation

func getMsg(fn: String, ln: String) -> String {
    return fn == "Linus" && ln == "Zhang" ? "牛逼" : "不牛逼"
}

print("What is your first name?")
let firstName: String! = readLine(strippingNewline: true)
print(firstName)

print("What is your last name?")
let lastName: String! = readLine(strippingNewline: true)

print(">", getMsg(fn: firstName, ln: lastName))

File: nodejs/nb.js
'use strict'

const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

rl.question('What is your first name ?\n', firstName => {
  rl.question('What is your last name ?\n', lastName => {
    console.log('>',
      firstName === 'Linus' && lastName === 'Zhang'
        ? '牛逼' : '不牛逼'
    )
    rl.close()
  })
})
